site/src/pages/components/combobox.explainer.mdx

```diff
@@ -46,66 +47,33 @@ Further highlighting the need for such a solution, a recent [Twitter poll](https
-
-A good analogy for what this document will result in looking like is the [Tabs research document.](https://open-ui.org/components/tabs.research.parts/)
+This document aims to provide a complete view of combo-box utilisation across the web to inform a potential proposal within Open UI. The goal of this document is to propose a very basic anatomy to simplify the initial shipment of `<combobox>` (or some other name) in the web platform. It will heavily build upon stylable `select` but provide a better accessible and extensible solution than that of an `input` bound with `datalist`.
 
 ## Anatomy
-
 ```
 <combobox>
-   <input type=selectlist>
-   <listbox>
-	<option>One</option>
-	<option>Two</option>
-   </listbox>
+  <input type="text"/>
+  <button><selectedoption></selectedoption></button>
+  <datalist>
+    <option>Create a merge commit</option>
+    <option>Squash and merge</option>
+    <option>Rebase and merge</option>
+  </datalist>
 </combobox>
 ```
-
 <ComboboxAnatomy />
 
-<div class="question">
-  <em>This anatomy would require `input` to be supported by invokertarget, see issue [#839](https://github.com/openui/open-ui/issues/839)  ?</em>
-</div>
-
-
-- `<combobox>`: The root container that encapsulates the entire `combobox` structure. It provides context for the interaction between the `input` and the `listbox`.
-  - Slots:
-    - `<input>`
-    - `<listbox>`
-  - Attributes:
-    - **search**: Indicates the combobox will actively fetch results based on user input.
-    - **filter**: Indicates the combobox will narrow down visible options based on user input.
-    - **multiple**: Allows multiple options to be selected (will be added in future revisions in alignment with `selectlist`).
-- `<input type=selectlist>`: This is where the user can type input or view the selected option. It acts as the trigger to display the listbox.
-  - Slots:
-    - **selectedoption** (optional): To display the currently selected option from the `listbox`. 
-      - Note: Currently input doesn't support slots
-- `<listbox>`: The wrapper that contains the `<option>`(s) and `<optgroup>`(s). If this part was not provided by the author, then `<selectlist>` will automatically create one.
-  - Slots:
-    - `<option>`
-    - `<optgroup>` (optional)
-- `<option>`: Represents an individual selectable item within the `listbox`.
-- `<optgroup>` (Optional): A container to group related options together.
-  - Slots:
-    - `<legend>`: To label the group.
-    - `<option>`: The options within this group.
-- `<legend>` (Optional, within `<optgroup>`): Provides a label or title for a group of options within the `listbox`.
+- `<combobox>`: The root container that encapsulates the entire `combobox` structure. It provides context for the interaction between the `input` and the `datalist`.
+- `<input type=text>`: This is where the user can type input or view the selected option. It acts as the trigger to display the datalist.
```

Comment:
> ... view the selected option

This isn't true per your anatomy. This is what `<selectedoption>` is for. I think a separate issue may be warranted that opens the can of worms that is "input taking complex content".